Sources/Commands/SwiftTestCommand.swift

```diff
@@ -325,7 +331,11 @@ public struct SwiftTestCommand: AsyncSwiftCommand {
                     result = runner.ranSuccessfully ? .success : .failure
                 }
 
-                try generateXUnitOutputIfRequested(for: testResults, swiftCommandState: swiftCommandState)
+                try generateXUnitOutputIfRequested(
+                    for: testResults,
+                    swiftCommandState: swiftCommandState,
+                    detailedFailureMessage: self.options.shouldShowDetailedFailureMessage
```

Comment:
`self.options` should be available within this function, no?

Reply:
Am I not using `self.options`?